ci/pipeline.yaml

```diff
@@ -0,0 +1,578 @@
+
+---
+resource_types:
+  - name: google-cloud-storage
+    type: docker-image
+    source:
+      repository: frodenas/gcs-resource
+  - name: codebuild
+    type: registry-image
+    source:
+      repository: cedricziel/concourse-codebuild-resource
+      tag: "0.1.14"
+
+resources:
+  - name: agent-operator-release-source
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/instana/instana-agent-operator.git
+      branch: main
+      #  match release tags like "v1.12.99"
+      #  match pre-release tags like "v1.12.100-pre"
+      tag_regex: ^v\d*\.\d*\.\d*.*$
+      #          ^ beginning of the tag string
+      #           ^ all tags start with "v"
+      #            ^ ^ any number of digits
+      #               ^ the "." character
+      #                 ^ ^ any number of digits
+      #                    ^ the "." character
+      #                      ^ ^ any number of digits
+      #                         ^^ any prefix (denoting a non-release tag)
```

Comment:
Love the description.
nitpick: shouldn't it be `\d+` everywhere? Because a `*` is zero to many, a `+` would ensure at least a single digit.